src/renderer/services/docker/chain.ts

```diff
@@ -0,0 +1,42 @@
+import {Container} from "./container";
+
+type LogType = "info" | "error";
+type LogCallbackFunc = (type: LogType, message: string) => void;
+
+export enum SupportedNetworks {
+    PRYSM = "Prysm",
+}
+
+export class BeaconChain extends Container {
+    public static instance: BeaconChain;
+
+    public static async startPrysmBeaconChain(waitUntilReady = false): Promise<BeaconChain> {
+        const bc = new BeaconChain({
+            image: "gcr.io/prysmaticlabs/prysm/beacon-chain:latest",
+            name: `${SupportedNetworks.PRYSM}-beacon-node`,
+            restart: "unless-stopped",
+            ports: ["4000:4000", "13000:13000"],
```

Comment:
I don't think port 13000 needs to be exposed to host.
Also user should be able to configure on which port does he wan't port 4000 to be exposed(aka this should be configurable by user)